brave-okhttp/README.md

```diff
@@ -0,0 +1,17 @@
+# brave-okhttp #
+
+Contributed by [Tristan Burch](https://github.com/tburch).
```

Comment:
No Problem. Just copied the readme from the http client module and changed things.